runescape-api/src/main/java/net/runelite/rs/api/RSClient.java

```diff
@@ -621,4 +621,16 @@
 
 	@Import("flags")
 	int getFlags();
+
+	@Import("compass")
+	void setCompass(SpritePixels spritePixels);
+
+	@Import("widgetSpriteCache")
+	RSNodeCache getWidgetSpriteCache();
+
+	@Override
+	void setSpriteOverrides(Map<Integer, SpritePixels> overrides);
```

Comment:
These two methods are not necessary here and generally we do not declare as part of rs-api methods implemented in mixins.